### PR TITLE
fix: analyze arrow variable functions

### PR DESCRIPTION
**Pull request by Fiszcz**

### CODE REVIEW

File: src/core/interaction-creator.ts [hunk 1]
```diff
@@ -150,6 +151,18 @@ export class InteractionCreator {
         }
     };
 
+    private static getNameOfFunction = (nodeWithFunction: tsMorph.Node): string => {
```
Comment (pawfa) [inline src/core/interaction-creator.ts:154]: What about getFunctionName? :)

**Review by pawfa — APPROVED** [review]

File: src/core/interaction-creator.ts (hunk 1 above)
Comment (Fiszcz) [reply src/core/interaction-creator.ts:154]: yeah, good idea

File: tests/int/testCases/different-functions/apiFunctions.ts [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+    /**
+     * @pact
+     * @pact-method GET
+     * @pact-path /api/property/standard
+     */
+    propertyStandardFunction: function () {
+        return 'string';
+    },
+};
```
Comment (lukaszfiszer) [inline tests/int/testCases/different-functions/apiFunctions.ts:37]: shorthand method

```
let o = {
  fun() {
     // ...
  },
}
```

is missing
```

**Review by lukaszfiszer — APPROVED** [review]

File: tests/int/testCases/different-functions/apiFunctions.ts (hunk 2 above)
Comment (Fiszcz) [reply tests/int/testCases/different-functions/apiFunctions.ts:46]: oks. Right
I've added

**Pull request merged**